_posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md

```diff
@@ -0,0 +1,111 @@
+---
+
+title: 'SPEED CODERS OFFER ALIEN ROBOTS SCHOLARSHIPS: TGIF(79)'
+author: Cecily
+layout: blog-post-tgif
+comments: false
+
+---
+
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday, and again it's time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. As usually, they're curated for you by [Cecily](https://twitter.com/skeskali)
+
```

Comment:
should we turn these two links into markdown links? Right now they're just text.